src/data/validation/email.js

```diff
@@ -155,19 +183,48 @@ const returnValidatedEmails = (formData) => {
   if (errorsDict._error.length > 0) {
     throw new SubmissionError(errorsDict);
   }
-  let emails = [];
+  let emails = extractEmails(formData).allEmails;
+  emails = _.union(emails); // Dedup emails
```

Comment:
nit: would `const emails = _.union(extractEmails(formData.allEmails)) // Extract and dedup emails` be logically equivalent without needing to define and then re-define `emails`?